### PR TITLE
12.0 mig pos remove pos category

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
@code review. No test. Thanks.

File: pos_remove_pos_category/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2015-TODAY Akretion (<http://www.akretion.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'POS Remove POS Category',
+    'version': '12.0.2.0.0',
+    'author': 'Akretion, Camptocamp SA, ACSONE SA/NV, '
+              'Odoo Community Association (OCA)',
+    'category': 'Sales Management',
```
Comment (MeetKD) [inline pos_remove_pos_category/__manifest__.py]: category should be -> Point of Sale

File: pos_remove_pos_category/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'depends': [
+        'point_of_sale',
+    ],
+    'demo': [],
```
Comment (MeetKD) [inline pos_remove_pos_category/__manifest__.py]: Can we remove empty key ?

File: pos_remove_pos_category/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    'website': 'https://www.akretion.com',
```
Comment (MeetKD) [inline pos_remove_pos_category/__manifest__.py]: Website -> https://github.com/OCA/pos/

File: pos_remove_pos_category/models/module.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                    UPDATE ir_act_window iaw SET res_model='pos.category'
+                    FROM ir_model_data imd
+                    WHERE
+                        iaw.id = imd.res_id AND
+                        imd.model = 'ir.actions.act_window' AND
+                        imd.name = 'product_pos_category_action'
+                ''')
+
+                break
+
```
Comment (MeetKD) [inline pos_remove_pos_category/models/module.py]: Can we remove the extra lines ?

File: pos_remove_pos_category/static/src/js/pos_remove_pos_category.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ ******************************************************************************/
+
+odoo.define('pos_remove_pos_category.remove_pos_category', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var pos_models = require('point_of_sale.models');
+    var pos_screens = require('point_of_sale.screens');
+    var _t = core._t;
```
Comment (MeetKD) [inline pos_remove_pos_category/static/src/js/pos_remove_pos_category.js]: I think _t is not used. We can remove this.

**Review by MeetKD — CHANGES_REQUESTED** [review]
Minor Changes

**Comment by sebastienbeau** [comment]
@MeetKD I just pushed the fixes

File: pos_remove_pos_category/static/src/js/pos_remove_pos_category.js (hunk 5 above)
Comment (sebastienbeau) [reply pos_remove_pos_category/static/src/js/pos_remove_pos_category.js]: done

File: pos_remove_pos_category/models/module.py (hunk 4 above)
Comment (sebastienbeau) [reply pos_remove_pos_category/models/module.py]: done

File: pos_remove_pos_category/__manifest__.py (hunk 3 above)
Comment (sebastienbeau) [reply pos_remove_pos_category/__manifest__.py]: done

File: pos_remove_pos_category/__manifest__.py (hunk 2 above)
Comment (sebastienbeau) [reply pos_remove_pos_category/__manifest__.py]: done

File: pos_remove_pos_category/__manifest__.py (hunk 1 above)
Comment (sebastienbeau) [reply pos_remove_pos_category/__manifest__.py]: done

**Review by hparfr — APPROVED** [review]
LGTM

**Review by MeetKD — APPROVED** [review]
Looks Good.

**Pull request merged**